src/commands/android/subcommands/install/index.ts

```diff
@@ -12,3 +26,20 @@ export async function install(options: Options, sdkRoot: string, platform: Platf
   return false;
 }
 
+async function flagsPrompt(options: Options) {
```

Comment:
function name should be an action verb.

Reply:
Also, we should ideally move this function to common.ts since the same function can be reused for this purpose. We can store all necessary information required for the prompt in `AVAILABLE_SUBCOMMANDS` itself.